state/state.go

```diff
@@ -2181,8 +2190,8 @@ func (st *State) AddRelation(eps ...Endpoint) (r *Relation, err error) {
 				ops = append(ops, txn.Op{
 					C:      remoteApplicationsC,
 					Id:     st.docID(ep.ApplicationName),
-					Assert: bson.D{{"life", Alive}},
-					Update: bson.D{{"$inc", bson.D{{"relationcount", 1}}}},
+					Assert: assertions,
+					Update: bson.D{{Name: "$inc", Value: bson.D{{Name: "relationcount", Value: 1}}}},
```

Comment:
I find it better to squint at all linting warnings around `bson.D` :smile:

Reply:
Make Tiobe Happy as a Service :) every little bit helps you increase that rating